coderd/database/postgres/postgres.go

```diff
@@ -39,9 +40,21 @@ func Open() (string, func(), error) {
 		}
 
 		dbName = "ci" + dbName
-		_, err = db.Exec("CREATE DATABASE " + dbName)
-		if err != nil {
-			return "", nil, xerrors.Errorf("create db: %w", err)
+		if os.Getenv("DB_FROM") == "" {
+			_, err = db.Exec("CREATE DATABASE " + dbName)
+			if err != nil {
+				return "", nil, xerrors.Errorf("create db: %w", err)
+			}
+
+			err = database.MigrateUp(db)
+			if err != nil {
+				return "", nil, xerrors.Errorf("migrate db: %w", err)
+			}
+		} else {
+			_, err = db.Exec("CREATE DATABASE " + dbName + " WITH TEMPLATE " + os.Getenv("DB_FROM"))
+			if err != nil {
+				return "", nil, xerrors.Errorf("create db with template: %w", err)
+			}
```

Comment:
I'm concerned that this function may return a database either with or without the migration depending on environment variables.  If the caller doesn't also run the migration, we'll have tests that succeed in CI, but fail locally, and that's no good.

Reply:
How would you propose we refactor this?

Reply:
could move the `MigrateUp()` call out of `coderdtest` entirely, and also include it in the arm this function where we start a docker container?  So, this function always gives you a fully migrated database.